src/Maps/TileSetSource.h

```diff
@@ -13,7 +14,7 @@ struct TilesetSource
 	std::string tilesetFilename;
 
 	// Number of Tiles in set (represented on BMP).
-	int numTiles;
+	uint32_t numTiles;
```

Comment:
Again, this struct contains a `string`, making it an in-memory struct, rather than a file format image struct. Hence the fixed size type is perhaps not strictly required here. It might actually make sense to use a `std::size_t`.